services/team.service.js

```diff
@@ -31,8 +28,6 @@ function findTeamByHackerId(hackerId) {
  * @description Adds a new team to database.
  */
 async function createTeam(teamDetails) {
-    const TAG = `[Team Service # createTeam]:`;
```

Comment:
A lot of unused TAGs, we should discuss whether we want to keep these, and if so, how to utilize them for debugging/logging purposes

Reply:
Yeah, we need to either remove, or use them. Your call.